### PR TITLE
Miser mode enhancements

### PR DESCRIPTION
**Pull request by stwalkerster**
Couple of basic improvements to miser mode considering the current state of play in production

* Change the colour of the badge in miser mode
* Add list all button for those who are insane enough to click it. See commit message for more info.

### CODE REVIEW

**Comment by stwalkerster** [comment]
Testing: https://phabricator.stwalkerster.co.uk/M3

File: acc.php [hunk 1]
```diff
@@ -2035,6 +2035,43 @@
 		}
 	});
 }
```
Comment (FastLizard4) [inline acc.php]: Minor code style: elseif should be on the same line as the closing brace of the previous block

File: acc.php [hunk 2]
```diff
@@ -2035,6 +2035,43 @@
...
+    if ($enableEmailConfirm == 1) {
+        $query = "SELECT * FROM request WHERE status = :type AND emailconfirm = 'Confirmed';";
+    }
+    else {
+        $query = "SELECT * FROM request WHERE status = :type;";
+    }
+
+    $statement = $database->prepare($query);
+
+    if (isset($_GET['status']) && isset($availableRequestStates[$_GET['status']])) {
```
Comment (FastLizard4) [inline acc.php]: Maybe it makes sense to move this conditional up to just after the `global $availableRequestStates`?  That way we don't have to spend time preparing the database statement unless we have everything we need.

File: acc.php (hunk 1 above)
Comment (stwalkerster) [inline acc.php]: All style fixes applied in newinternal branch. 😉

Comment (FastLizard4) [inline acc.php]: fien

**Comment by FastLizard4** [comment]
good enough™

**Pull request merged**